### ISSUE TITLE
Some features suggest

### ISSUE THREAD

**Issue by CrazyWords1** [SkriptLang/Skript#1824]
### Description
**I suggest adding a few features**
- [ ] Menu manager
- [ ] Scoreboard 
- [X] ActionBar
- [ ] BossBar 
- [ ] Yml files support

P.S (The fact is that my server is almost entirely built on Script without any Addons and I do not want to put large Addons, for simple functions)

**Comment by TheBentoBox** [comment]
* Menu manager: do you mean something like TuSKe? If so you can already do most of that in vanilla Skript via all of the existing inventory support, it's just less directly convenient than in TuSKe.
* [Action bars are already supported.](http://skriptlang.github.io/Skript/effects#EffActionBar)
* For the rest it's up to the devs but I wouldn't be surprised if they dislike the idea of YML and scoreboard support. That stuff is more involved and not really fitting for vanilla Skript, especially scoreboards. Boss bars would make enough sense though.

**Comment by ShaneBeee** [comment]
I personally like the Idea of YAML support for sure. Since it is something that in Spigot is used by virtually EVERY plugin, it would be super handy to have it built in.

I personally think that scoreboards don't really fit into Skript as they're super complicated.

**Comment by Pikachu920** [comment]
personally I'd rather not see YAML in skript. as for scoreboards, they are fine but the issue is they are tough to get right and require NMS for advanced features

**Comment by Snow-Pyon** [comment]
Scoreboards could be implemented at some point but it's a tough task just as Pikachu said. As for YML, that's a strong no from my part, I wouldn't accept PRs on it either.

One of the reasons is that the main point of Skript as plugin is supporting everything MC-related, YML isn't MC-related and thus I'd say it belongs to an addon (and there are addons that handle it pretty well already).

**Comment by andrei923** [comment]
I think a format slot effect will be really useful.

**Comment by bensku** [comment]
Consider splitting many feature requests to many issues in future.

**Comment by TrademarkTM** [comment]
I agree with @andrei923, adding GUI syntaxes similar to TuSKe into Skript would be quite handy.

As for scoreboards, I don't see a reason to implement it into Skript. There are plenty of good plugins that handle that quite well (TitleManager, QuickBoard, Featherboard). For me, hooking Skript into them would make more sense (something like Skore addon). Bossbars are a nice feature, though.

**Comment by Snow-Pyon** [comment]
Scoreboards aren't just sidebar displays, it has many uses besides that.

As for the GUI syntaxes, we're better off letting an addon make them or people to make snippets for it, that's what I think at least.

**Comment by AyhamAl-Ali** [comment]
There is unfinished code for Bossbar implementation in #3995

**Comment by sovdeeth** [comment]
Closing as this isn't a very useful issue
